### ISSUE TITLE
watch function throws TypeError under certain conditions

### ISSUE THREAD

**Issue by h-sakano** [react-hook-form/react-hook-form#2246]
**Describe the bug**

'watch' throws TypeError when the `defaultValue` of the Controller component is `null`.
This issue has occurred since 6.0.6 and not until 6.0.5.

**To Reproduce**

1. Go to https://codesandbox.io/s/react-hook-form-usefieldarray-template-ngkqb
2. Click on 'append'
3. See error

**Codesandbox link (Required)**

https://codesandbox.io/s/react-hook-form-usefieldarray-template-ngkqb

**Screenshots**

![TypeError](https://user-images.githubusercontent.com/26650149/87618519-0c41ae00-c755-11ea-86a6-639a491a6f70.png)

**Comment by h-sakano** [comment]
I'm sorry. I just noticed that it is a duplicate of #2239 🙇‍♂️

**Comment by bluebill1049** [comment]
Thanks! I will close the other one, as this one has more detail.

**Comment by bluebill1049** [comment]
could you please verify the fix is working: https://codesandbox.io/s/react-hook-form-usefieldarray-template-zkow9?file=/src/index.js:79-84?

**Comment by h-sakano** [comment]
> could you please verify the fix is working: https://codesandbox.io/s/react-hook-form-usefieldarray-template-zkow9?file=/src/index.js:79-84?

It works perfectly 🎉 
Thank you for your great work!!

**Comment by bluebill1049** [comment]
Thanks very much, @h-sakano for the reporting and detailed issue. I will follow it up with some unit test and release the patch soon.
